src/client/render/interlaced.cpp

```diff
@@ -35,7 +35,11 @@ void RenderingCoreInterlaced::initMaterial()
 	IShaderSource *s = client->getShaderSource();
 	mat.UseMipMaps = false;
 	mat.ZBuffer = false;
+#if (IRRLICHT_VERSION_MAJOR > 1 || IRRLICHT_VERSION_MINOR >= 9) && !defined(__ANDROID__)
```

Comment:
If the newer Irrlicht works fine Android should be made to use it too instead of putting a `&& !defined(__ANDROID__)` everywhere

(also I'd prefer if the Irrlicht support changes were made in a separate PR)

Reply:
IMO updating the version of Irrlicht used for Android builds should be a separate PR, since the Android version uses its own fork. This probably should be a separate PR though, it was just easier to test when all the changes were in one branch, since the Irrlicht patch is based on the latest revision.

Reply:
Now that everything uses Minetest's Irrlicht, this isn't relevant anymore.